www/templates/index.php

```diff
@@ -35,44 +28,92 @@
+                            <input class="ga-required" id="gaName" name="gaName" type="text" aria-labelledby="gaNameLabel" aria-describedby="gaNameDesc">
+                            <span class="dcf-form-help" id="gaNameDesc" tabindex="-1">Product, promo code or slogan</span>
+                        </span>
+                    </div>
+                    <div class="dcf-form-group">
+                        <label id="gaMediumLabel" for="gaMedium">Medium <small class="dcf-required">Required</small></label>
+                        <span>
+                            <input class="ga-required" id="gaMedium" name="gaMedium" type="text" aria-labelledby="gaMediumLabel" aria-describedby="gaMediumDesc">
+                            <span class="dcf-form-help" id="gaMediumDesc" tabindex="-1">Marketing medium: email, web, banner</span>
+                        </span>
+                    </div>
+                    <div class="dcf-form-group">
+                        <label id="gaSourceLabel" for="gaSource">Source <small class="dcf-required">Required</small></label>
+                        <span>
+                            <input class="ga-required" id="gaSource" name="gaSource" type="text" aria-labelledby="gaSourceLabel" aria-describedby="gaSourceDesc">
+                            <span class="dcf-form-help" id="gaSourceDesc" tabindex="-1">Referrer: Google, Facebook, Twitter</span>
+                        </span>
+                    </div>
+                    <div class="dcf-form-group">
+                        <label id="gaTermLabel" for="gaTerm">Term</label>
+                        <span>
+                            <input id="gaTerm" name="gaTerm" type="text" aria-labelledby="gaContentLabel" aria-describedby="gaTermDesc">
+                            <span class="dcf-form-help" id="gaTermDesc" tabindex="-1">Identify the keywords</span>
+                        </span>
+                    </div>
+                    <div class="dcf-form-group">
+                        <label id="gaContentLabel" for="gaContent">Content</label>
+                        <span>
+                            <input id="gaContent" name="gaContent" type="text" aria-labelledby="gaContentLabel" aria-describedby="gaContentDesc">
+                            <span class="dcf-form-help" id="gaContentDesc" tabindex="-1">Use to differentiate ads (A/B testing)</span>
+                        </span>
+                    </div>
                 </div>
             </fieldset>
             <input class="dcf-mt-6" id="submit" name="submit" type="submit" value="Create URL">
         </div>
     </form>
 </div>
+
+<!--Script for displaying Google Analytics Campaign Tagging when checkbox is clicked-->
+<script>
+    var withGACheckbox = document.getElementById('with-ga-campaign');
+    var gaSection = document.getElementById('ga-tagging');
+    var gaRequiredVars = document.getElementsByClassName('ga-required');
+
+    withGACheckbox.addEventListener('click', function(){
```

Comment:
Please correct the conditional code style by adding a space so `function(){` is `function() {`